### DISCUSSION TITLE
solving lyapunov function using Sum of squares

### DISCUSSION THREAD

**Discussion by Mahmoud-shabansource** [yalmip/YALMIP#1136] [Optimization modelling] [answered]
I want to find Lyapunov function in this paper "An SOS-based Observer Design for Polynomial Fuzzy Systems"
  for different degrees as in the paper, but i don't know How. could you please help?
thanks
![image](https://user-images.githubusercontent.com/107326134/173785173-ce242543-6feb-4721-8e25-2611cc29a329.png)

**Comment by johanlofberg** [comment, marked as answer]
You would have to be more specific in your questions
https://yalmip.github.io/tutorial/sumofsquaresprogramming/
